### PR TITLE
Feature/as 3619 link lpa questionnaire pages

### PR DESCRIPTION
**Pull request by nssdesikan**
These changes are related pulling the information from the SQL database and Document service API

### CODE REVIEW

File: packages/web-app/src/routes/index.spec.js [hunk 1]
```diff
@@ -52,12 +52,9 @@ describe('routes/index', () => {
...
+      `/questionnaire-check-and-confirm`,
       questionnaireCheckAndConfirm
     );
-    expect(mockUse).toBeCalledWith(
-      `/questionnaires-for-review/already-reviewed`,
-      questionnaireAlreadyReviewed
-    );
+    expect(mockUse).toBeCalledWith(`/questionnaire-already-reviewed`, questionnaireAlreadyReviewed);
   });
 });
```
Comment (swdpcomputing) [inline packages/web-app/src/routes/index.spec.js:60]: I can't see a test for reviewQuestionnaireSubmission on this page.

File: packages/web-app/src/middleware/check-appeal-outcome.spec.js [hunk 2]
```diff
@@ -18,10 +18,7 @@ describe('middleware/check-appeal-outcome', () => {
       title: 'Return next if there is appeal id in the params',
       req: {
         ...mockReq,
-        session: {},
-        params: {
-          appealId: '1',
-        },
+        session: { appeal: { appealId: 'b1a04ba7-9604-4196-b6b4-b16b4acd1875' } },
```
Comment (swdpcomputing) [inline packages/web-app/src/middleware/check-appeal-outcome.spec.js:21]: Why is there a hard-coded uuid for lines 21, 32, 45?

File: packages/web-app/src/middleware/check-appeal-outcome.spec.js [hunk 3]
```diff
@@ -18,10 +18,7 @@ describe('middleware/check-appeal-outcome', () => {
       title: 'Return next if there is appeal id in the params',
```
Comment (swdpcomputing) [inline packages/web-app/src/middleware/check-appeal-outcome.spec.js:18]: It doesn't appear that you're using params anymore...

File: packages/web-app/src/middleware/check-appeal-outcome.spec.js [hunk 4]
```diff
@@ -32,10 +29,8 @@ describe('middleware/check-appeal-outcome', () => {
       req: {
```
Comment (swdpcomputing) [inline packages/web-app/src/middleware/check-appeal-outcome.spec.js:29]: How does this test if the review oputcome is complete? check-appeal-outcome.js does not set any outcome anymore.

File: packages/web-app/src/views/components/files-cell.njk [hunk 5]
```diff
@@ -1,17 +1,19 @@
 {% macro filesCell(data) %}
```
Comment (swdpcomputing) [inline packages/web-app/src/views/components/files-cell.njk:2]: DD is the advised pattern in gds. Why is it being changed here?

**Review by swdpcomputing — CHANGES_REQUESTED** [review]

File: packages/web-app/src/middleware/check-appeal-outcome.spec.js (hunk 4 above)
Comment (nssdesikan) [reply packages/web-app/src/middleware/check-appeal-outcome.spec.js:29]: They don't set the outcome but these tests are to ensure the middleware calls next when the _outcome_ is set

File: packages/web-app/src/views/components/files-cell.njk (hunk 5 above)
Comment (nssdesikan) [reply packages/web-app/src/views/components/files-cell.njk:2]: Changed to show list of files as a list rather as in the prototype

File: packages/web-app/src/middleware/check-appeal-outcome.spec.js (hunk 4 above)
Comment (swdpcomputing) [reply packages/web-app/src/middleware/check-appeal-outcome.spec.js:29]: OK

File: packages/web-app/src/views/components/files-cell.njk (hunk 5 above)
Comment (swdpcomputing) [reply packages/web-app/src/views/components/files-cell.njk:2]: The prototype is just a visual representation, we ah supposed to go with gds.

File: packages/web-app/src/routes/index.spec.js (hunk 1 above)
Comment (nssdesikan) [reply packages/web-app/src/routes/index.spec.js:66]: Addressed via commit [0a79768](https://github.com/Planning-Inspectorate/back-office/pull/83/commits/0a797681b995faa4d2229ae2b0a99b192eaa6f5d)

File: packages/web-app/src/middleware/check-appeal-outcome.spec.js (hunk 3 above)
Comment (nssdesikan) [reply packages/web-app/src/middleware/check-appeal-outcome.spec.js]: Addressed via commit [0a79768](https://github.com/Planning-Inspectorate/back-office/pull/83/commits/0a797681b995faa4d2229ae2b0a99b192eaa6f5d)

**Pull request merged**